lib/hacluster.pm

```diff
@@ -618,7 +618,15 @@ sub ha_export_logs {
     upload_logs($mdadm_conf, failok => 1);
 
     # supportconfig
-    script_run "supportconfig -g -B $clustername", 300;
+    my $ret = script_run "supportconfig -g -B $clustername", 300, die_on_timeout => 0;
+    # Make it softfail for not blocking qem bot auto approvals on 12-SP5
+    # Command 'supportconfig' hangs on 12-SP5, script_run timed out and returned 'undef'
+    # Currently not sure if it is related to poo#151612
+    if (!defined($ret) && is_sle("=12-SP5")) {
+        record_soft_failure 'jsc#TEAM-8837';
```

Comment:
Now that we've confirmed that this was impacted by the communication issue to external services hosted on AWS, I think it's better to softfail with `poo#151612` instead of a team ticket. Otherwise it LGTM.